src/CovidAppointmentTable.js

```diff
@@ -15,6 +16,7 @@ import SignUpLink, { hasSignUpLink } from "./components/SignUpLink";
 import StaleDataIndicator from "./components/StaleDataIndicator";
 import Switch from "@material-ui/core/Switch";
 import Typography from "@material-ui/core/Typography";
+import { AlertTitle } from "@material-ui/lab";
```

Comment:
instead, please do
```
import AlertTitle from '@material-ui/lab/AlertTitle';
```
doing that kind of import saves space when things get bundled for production (IIRC)
(and then place alphabetically)